src/styles/components/_card.scss

```diff
@@ -0,0 +1,164 @@
+@import "../abstracts/_variables.scss";
+@import "../base/base";
+
+@media only screen and (max-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(1, 1fr);
+    max-width: $card-width;
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, 1fr);
+    max-width: calc(2 * #{$card-width} + 1 * #{$horizontal-gap});
```

Comment:
you don't want to calculate such things by hand, imagine that your design has 10 different resolution.